phase_unwrap.go

```diff
@@ -29,14 +30,31 @@ func NewPhaseUnwrapper(fractionBits, lowBitsToDrop uint) *PhaseUnwrapper {
 	u.lowBitsToDrop = lowBitsToDrop
 	u.twoPi = int16(1) << (fractionBits - lowBitsToDrop)
 	u.onePi = u.twoPi >> 1
-	u.resetAfter = 20000 // TODO: this should be a settable parameter
+	u.resetAfter = resetAfter
+	u.enable = enable
+	if resetAfter <= 0 {
```

Comment:
This is silent error handling. Do not do silent error handling. Just panic and throw an error in the relevant RPC function. Do not silently change user input parameters.

Reply:
Well, one could just as easily view this as enforcing consistency. That is, non-positive values here are taken to _mean_ disabled phase unwrapping.

However, I admit that this would not be the most obvious interpretation. I think a safer procedure might be to panic in case of inconsistency here, but to enforce consistency in the RPC server so that within Dastard the panic can't actually happen. I'll do that.